lib/kvdb/ikvdb.c

```diff
@@ -1791,6 +1783,14 @@ ikvdb_get_csched(struct ikvdb *handle)
     return handle ? ikvdb_h2r(handle)->ikdb_csched : 0;
 }
 
+struct kvdb_rparams *
+ikvdb_get_rp(struct ikvdb *ikvdb)
```

Comment:
Isn't there already an API for this? If not, spell out rparams, and maybe return a const view

Reply:
I couldn't find one, was surprised..